Create makefile for installing pre-built binaries and pkg-config file
--- Makefile	1970-01-01 01:00:00.000000000 +0100
+++ Makefile	2016-11-10 17:31:08.686722661 +0000
@@ -0,0 +1,21 @@
+all:	libomegonprocam.pc
+
+%.pc:	%.pc.in
+	sed -e "s!@LIBDIR@!$(LIBDIR)!g" \
+		-e "s!@VERSION@!$(VERSION)!g" < $< > $@
+
+install:
+	mkdir -p $(LIBDIR) $(INCDIR) $(PKGCONFDIR) $(UDEVDIR)
+	case $(DEB_BUILD_ARCH) in \
+	amd64) \
+		cp $(SRCDIR)/linux/x64/libomegonprocam*.so.$(VERSION) $(LIBDIR);; \
+	aarch64|arm64) \
+		cp $(SRCDIR)/linux/arm64/libomegonprocam*.so.$(VERSION) $(LIBDIR);; \
+	armhf) \
+		cp $(SRCDIR)/linux/armhf/libomegonprocam*.so.$(VERSION) $(LIBDIR);; \
+	esac
+	ln -sf libomegonprocam.so.$(VERSION) $(LIBDIR)/libomegonprocam.so.$(MAJOR_REV)
+	ln -sf libomegonprocam.so.$(VERSION) $(LIBDIR)/libomegonprocam.so
+	cp $(SRCDIR)/inc/*.h $(INCDIR)
+	cp $(SRCDIR)/libomegonprocam.pc $(PKGCONFDIR)
+	cp $(SRCDIR)/70-omegonpro-cameras.rules $(UDEVDIR)
